### PR TITLE
[DOCS] Corrected API path for invalidate token and SSL certificate examples

### PR DESCRIPTION
**Pull request by lcawl**
Related to https://github.com/elastic/elasticsearch/pull/39517

This PR corrects some out-dated examples in the invalidate token API and the SSL certificate API.
It also wraps some long lines of text.

Ideally we can enable the tests on these APIs so that we catch changes like this.

### CODE REVIEW

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 1]
```diff
@@ -31,25 +32,52 @@ The following parameters can be specified in the body of a DELETE request and
...
-or `refresh_token` parameters is required. If none of these two are specified, then `realm_name` and/or `username`
-need to be specified.
+NOTE: While all parameters are optional, at least one of them is required. More
+specifically, either one of `token` or `refresh_token` parameters is required.
+If none of these two are specified, then `realm_name` and/or `username` need to
+be specified.
 
 ==== Examples
 
+For example, if you create the following token:
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]:
```suggestion
For example, if you create a token using the 
`client_credentials` `grant_type` as follows:
```

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 2]
```diff
@@ -31,25 +32,52 @@ The following parameters can be specified in the body of a DELETE request and
...
+[source,js]
+--------------------------------------------------
+POST /_security/oauth2/token
+{
+  "grant_type" : "client_credentials"
+}
+--------------------------------------------------
+// CONSOLE
+
+The API returns the following information about the access token:
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]:
```suggestion
The Get Token API returns the following information about the access token:
```

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 3]
```diff
@@ -31,25 +32,52 @@ The following parameters can be specified in the body of a DELETE request and
...
+--------------------------------------------------
+{
+  "access_token" : "dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==",
+  "type" : "Bearer",
+  "expires_in" : 1200
+}
+--------------------------------------------------
+// TESTRESPONSE[s/dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==/$body.access_token/]
+
+
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]: Suggestion: 

This access token can now be immediately invalidated, as shown in the following example:

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 4]
```diff
@@ -59,57 +87,94 @@ DELETE /_security/oauth2/token
   "token" : "dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ=="
 }
 --------------------------------------------------
-// NOTCONSOLE
+// CONSOLE
+// TEST[s/dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==/$body.access_token/]
+// TEST[continued]
+
+If you used the `password` grant type to obtain a token for a user, the response
+contains a refresh token. For example:
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]:
```suggestion
might also contain a refresh token. For example:
```

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 5]
```diff
@@ -59,57 +87,94 @@ DELETE /_security/oauth2/token
...
+  "grant_type" : "password",
+  "username" : "test_admin",
+  "password" : "x-pack-test-password"
+}
+--------------------------------------------------
+// CONSOLE
+// TEST
 
-whereas the following example invalidates the specified refresh token immediately:
+The API returns the following information:
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]:
```suggestion
The Get Token API returns the following information:
```

File: x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc [hunk 6]
```diff
@@ -59,57 +87,94 @@ DELETE /_security/oauth2/token
...
+  "access_token" : "dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==",
+  "type" : "Bearer",
+  "expires_in" : 1200,
+  "refresh_token": "vLBPvmAB6KvwvJZr27cS"
+}
+--------------------------------------------------
+// TESTRESPONSE[s/dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==/$body.access_token/]
+// TESTRESPONSE[s/vLBPvmAB6KvwvJZr27cS/$body.refresh_token/]
+
+You can invalidate the specified refresh token as follows:
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc]:
```suggestion
The refresh token can now also be immediately invalidated as shown 
in the following example:"
```

**Review by jkakavas — COMMENTED** [review]
I left some suggestions

**Comment by lcawl** [comment]
The DELETE examples in the invalidate token API reference don't work in the Kibana Console. They return:

> {
  "error": {
    "root_cause": [
      {
        "type": "parse_exception",
        "reason": "request body is required"
      }
    ],
    "type": "parse_exception",
    "reason": "request body is required"
  },
  "status": 400
}

This is related to https://github.com/elastic/kibana/issues/13245

Thus I've marked those examples as //KIBANA to only show the "Copy as curl" link.

**Comment by lcawl** [comment]
Re-enabling //CONSOLE testing now that https://github.com/elastic/kibana/pull/32407 is merged.

**Comment by lcawl** [comment]
In case it's helpful, these changes can be previewed here:
https://doc-review-63bf6.firebaseapp.com/es2/security-api-invalidate-token.html

**Review by jkakavas — APPROVED** [review]
LGTM

**Pull request merged**